README.md

```diff
@@ -304,6 +304,20 @@ address("/price/usd/gbp")
   .post({foo:"bar"})
 ```
 
+### Adding timeout
+When we request for a resource, it waits for a specific time before it sends "408 - request-timeout" error. By adding timeout, we can increase the wait time for the requested resource.
```

Comment:
Let's make this read better:

> When requesting a resource  address will by default wait 30 seconds for a response from the resource, before responding with a `408 Request Timeout` error. By calling the `timeout` method on the request, it is possible to increase or decrease this timeout period. The following example shows how to increase the timeout from the default 30 seconds to 60:
> ```javascript
> // increase the timeout to 60000ms (60 sec) for request to resolve
> address("/price/usd/gbp")
>   .timeout(60000)
>   .json()
>   .get()
> ```
>
> The value given to the `timeout` method is in milliseconds.

I'm also not sure whether milliseconds makes sense for this kind of API. The minimum realistic timeout you could set would lie in the ~5-10ms range, and anything sub-second seems rather pointless to be honest. Should we just change this API to accept seconds insteald of milliseconds? That would obviously mean the default would be set to `30` instead of `30000` and the example above would use `60` instead of `60000`.

Reply:
Existing libraries, like d3, jQuery etc. are using time in milliseconds and also a standard way to provide duration in timeouts.